test-result-summary-client/src/Dashboard/Widgets/Graph/Dacapo.jsx

```diff
@@ -111,14 +113,52 @@ export default class Dacapo extends Component {
 				visible: key === "h2Data",
 				name: key,
 				data: series[key],
-				keys: ['x', 'y', 'additionalData', 'CI']
+				keys: ['x', 'y', 'additionalData', 'CI'],
+				events: {
+					click: async function (event) {
```

Comment:
Please move this in a common file (i.e.,  https://github.com/adoptium/aqa-test-tools/blob/master/test-result-summary-client/src/Dashboard/Widgets/Graph/utils.js) as this will be used for all perf widget.